src/itemChoice/index.tsx

```diff
@@ -40,96 +50,92 @@ export interface ItemChoiceProps {
   readonly onMouseDown?: (event: React.MouseEvent<HTMLElement>) => void,
 }
 
-class ItemChoice extends PureComponent <ItemChoiceProps> {
+class ItemChoice extends PureComponent<ItemChoiceProps> {
   static defaultProps: Partial<ItemChoiceProps> = {
+    disabled: false,
     highlighted: false,
     selected: false,
     status: ItemChoiceStatus.DEFAULT,
     href: '',
   }
+
   static STATUS = ItemChoiceStatus
+
   render() {
     const {
-      children, className, highlighted, selected, status,
-      onClick, onBlur, onFocus, onMouseDown, href, label, subLabel, leftAddon, rightAddon,
+      className,
+      highlighted,
+      selected,
+      disabled,
+      status,
+      onClick,
+      onBlur,
+      onFocus,
+      onMouseDown,
+      href,
+      label,
+      subLabel,
+      leftAddon,
+      rightAddon,
       onDoneAnimationEnd,
     } = this.props
-    const classNames = cc([prefix({
-      itemChoice: true,
-      'itemChoice--highlighted': highlighted,
-      'itemChoice--withSubLabel': !!subLabel,
-    }), className])
 
-    let rightIcon = <ChevronIcon className={cc(prefix({ chevron: true }))} />
+    const classNames = cc([
+      {
+        'kirk-itemChoice': true,
+        'kirk-itemChoice--highlighted': highlighted,
+        'kirk-itemChoice--disabled': disabled,
+      },
+      className,
+    ])
 
-    if (status === ItemChoiceStatus.LOADING) {
-      rightIcon = <Loader
-        className={cc(prefix({ chevron: true }))}
-        size={24}
-        onDoneAnimationEnd={onDoneAnimationEnd}
-        inline
-      />
-    }
-
-    if (status === ItemChoiceStatus.CHECKED) {
-      rightIcon = <Loader
-        className={cc(prefix({ chevron: true }))}
-        size={24}
-        onDoneAnimationEnd={onDoneAnimationEnd}
-        done
-        inline
-      />
-    }
+    const chevronDisplay =
+      status === ItemChoiceStatus.DEFAULT ? (
+        <ChevronIcon />
+      ) : (
+        <Loader
```

Comment:
When this is loading, shouldn't we put the itemChoice to disabled?

Reply:
UP :)

Reply:
Disable state is now implemented, so I guess I can do that yeah